src/isIsogram.test.js

```diff
@@ -6,4 +6,40 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for an empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should return true for "playgrounds"', () => {
```

Comment:
It is better to describe, not that it should return true for some exact word, but why it should return true?
Example:
'should return true if entered string has no repeatings'
Please review the rest of the tests.